rotkehlchen/api/v1/schemas.py

```diff
@@ -1332,6 +1333,10 @@ class ModifiableSettingsSchema(Schema):
     )
     auto_delete_calendar_entries = fields.Boolean(load_default=None)
     auto_create_calendar_reminders = fields.Boolean(load_default=None)
+    sync_method_upon_size_discrepancy = SerializableEnumField(
```

Comment:
Using an enum is not a bad idea, but since at the moment it's just 2 choices you can just use a boolean.